yt_dlp/postprocessor/embedthumbnail.py

```diff
@@ -90,12 +90,31 @@ def run(self, info):
 
         success = True
         if info['ext'] == 'mp3':
-            options = [
-                '-c', 'copy', '-map', '0:0', '-map', '1:0', '-write_id3v1', '1', '-id3v2_version', '3',
-                '-metadata:s:v', 'title="Album cover"', '-metadata:s:v', 'comment=Cover (front)']
+            # Method 1: Use mutagen
+            # Prioritize mutagen over ffmpeg since ffmpeg messes up the lyrics data
+            if mutagen:
+                try:
+                    self._report_run('mutagen', filename)
+                    audio = mutagen.id3.ID3(filename)
+                    if 'APIC' in audio:
+                        del audio['APIC']
+                    with open(thumbnail_filename, 'rb') as thumbfile:
+                        audio['APIC'] = mutagen.id3.APIC(
+                            encoding=3, mime='image/%s' % thumbnail_ext, type=3,
+                            desc=u'Cover (front)', data=thumbfile.read())
```

Comment:
```suggestion
                    with open(thumbnail_filename, 'rb') as thumbfile:
                        data = thumbfile.read()
                    audio['APIC'] = mutagen.id3.APIC(
                        mutagen.id3.Encoding.UTF8, f'image/{thumbnail_ext}',
                        mutagen.id3.PictureType.COVER_FRONT, 'Cover (front)', data)
```

Reply:
why?

Reply:
- `u'Cover (front)'` is legacy
- `'image/%s' % thumbnail_ext` can just be `f'image/{thumbnail_ext}`
- `type=3` might not be consistent, `mutagen.id3.PictureType.COVER_FRONT` is more explicit